---
title: "Oyster Introgression"
author: "L. Auckett"
date: "24/05/2021"
output:
  
  pdf_document: default
  html_document:
    df_print: kable
  word_document: default
  latex_engine: xelatex
  always_allow_html: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```

# Introduction and Methods 

This report hypothesises that gene flow is resulting in introgression is occurring between wild and selectively bred oysters that share an estuary, and that this introgression is altering the genome of the selectively bred populations.  

### Independent variables:
1. Allele frquency in wild oyster populations

### Dependent variables:
2. Allele frequency selectively bred oyster populations

The null hypothesis is that there will be no gene flow between wild and selectively bred populations of oysters, and as a result there will be two genetically distinct population clusters shown in the Principal Components Analysis. Assumptions of PCA include multiple variables at continuous levels, a linear relationship between all variables, no significant outliers, sampling adequacy, and adequate correlations suitable for data reduction. 
A paired t-test was performed to determine whether or not the mean of expected versus observed heterozygosities was significantly different to zero (assumes the differences between pairs are normally distributed).
A Bartlett test of homogeneity of variances was also performed (assumes dependent variable is normally distributed) with the null hypothesis being that there is no difference in variances. 
Observed heterozygosity versus expected heterozygosity per locus was also tested for, as was Hardy-Weinberg equilibrium (no mutation, random mating, no gene flow, infinite population size, and no selection). Expected heterozygosity was plotted as a function of observed heterozygosity. 

# Results 
```{r Oyster packages, include=FALSE}
library(usethis)
library(hierfstat)
library(ggplot2)
library(diveRsity)
library(adegenet)
library(plotly)
library(poppr)
library(pegas)
library(ade4)
library(xtable)
library(knitr)
library(kableExtra)


```

```{r Oyster readfile, include=FALSE}
MyData <- read.genepop("../oysterdata.GEN", ncode = 2)
```

```{r Oyster Introgression, include = FALSE}
# tab(MyData)
# summary(MyData)
div <- summary(MyData)
div
```

The null hypothesis is retained. There is no gene flow evident between the two populations. The PCA shows there are two distinct population clusters of oysters, one of selectively bred and one of wild. Populations within these clusters overlap but the two clusters themselves have no overlap.
Bartlettâ€™s test showed that data follows a normal distribution (p = 0.5642, df = 1, K-squared = 0.333).


```{r t test, echo=FALSE}
t.test(div$Hexp,div$Hobs,pair=T,var.equal=TRUE,alter="greater")
```


```{r poppr, include=FALSE}
poppr(
  MyData,
  total = TRUE,
  sublist = "ALL",
  exclude = NULL,
  blacklist = NULL,
  sample = 0,
  method = 1,
  missing = "ignore",
  cutoff = 0.05,
  quiet = FALSE,
  clonecorrect = FALSE,
  strata = 1,
  keep = 1,
  plot = TRUE,
  hist = TRUE,
  index = "rbarD",
  minsamp = 10,
  legend = FALSE)

```




``````{r Ho He, echo=FALSE}
plot(div$Hobs, div$Hexp, xlab="Observed Heterozygosity", ylab="Expected Heterozygosity")


```

``` {r pca1, include=FALSE}
sum(is.na(MyData$tab))
X <- tab(MyData, freq = TRUE, NA.method = "mean")
class(X)
dim(X)
X[1:5,1:5]
pca1 <- dudi.pca(X, scale = FALSE, scannf = FALSE, nf = 3)
barplot(pca1$eig[1:50], main = "PCA eigenvalues", col = heat.colors(50))

```

``` {r pca2, include=FALSE}
s.label(pca1$li)
title("PCA of Oyster dataset\naxes 1-2")
add.scatter.eig(pca1$eig[1:20], 3,1,2)
pca1
```

``` {r pca3, include = FALSE}
s.class(pca1$li, pop(MyData))
title("PCA of Oyster dataset\naxes 1-2")
add.scatter.eig(pca1$eig[1:20], 3,1,2)
```
``` {R scatter, include = FALSE}
s.class(pca1$li,pop(MyData),xax=1,yax=3,sub="PCA 1-3",csub=2)
title("PCA of Oyster dataset\naxes 1-3")
add.scatter.eig(pca1$eig[1:20],nf=3,xax=1,yax=3)

``` {r PCA scat, echo = FALSE}
col <- funky(15)
s.class(pca1$li, pop(MyData),xax=1,yax=3, col=transp(col,.6), axesell=FALSE,
        cstar=0, cpoint=3, grid=FALSE)


```
**Table 1** Mean and standard deviation of Observed Heterozygosity per population
``` {r include = FALSE}
MySummary<-basic.stats(MyData)
SummaryHo <- MySummary$Ho

SummaryHs <- MySummary$Hs

SummaryFis <- MySummary$Fis
SummaryFis <- SummaryFis[,-9] 
colMeans(SummaryFis,na.rm = TRUE)

print.df <- data.frame(Population=names(apply(SummaryHo,2,sd)),
                      Mean=round(apply(SummaryHo,2,mean),3),
                      SD=round(apply(SummaryHo,2,sd),3))
print.df1 <- data.frame(Population=names(apply(SummaryHs,2,sd)),
                       Mean=round(apply(SummaryHs,2,mean),3),
                       SD=round(apply(SummaryHs,2,sd),3))
print.df2 <- data.frame(Population=names(apply(SummaryFis,2,sd)),
                       Mean=round(apply(SummaryFis,2,mean),3),
                       SD=round(apply(SummaryFis,2,sd),3))
```

``` {r echo = FALSE}


kable(list(print.df, print.df1), row.names = FALSE)
kable(print.df2,row.names=FALSE, caption ="Fis")
```

``` {r include = FALSE}

clusterdata <- read.genepop("../ByCluster.GEN", ncode = 2)

head(clusterdata)
summary(clusterdata)
names(clusterdata)
clusterdata$pop
clustersummary <- basic.stats(clusterdata)

clustersummary$Ho
Ho <- colMeans(clustersummary$Ho)
Hs <- colMeans(clustersummary$Hs)
Fis <- colMeans(clustersummary$Fis, na.rm = TRUE)
means <- cbind(Ho, Hs, Fis)
```

``` {r echo = FALSE}
kable(means,row.names=TRUE, caption ="Means by cluster")
```

``` {r echo = FALSE}
PCA2 <- indpca(clusterdata) 
plot(PCA2, cex = 0.7, col="Coral2")
```

``` {r echo = FALSE}

qqnorm((div$Hexp - div$Hobs)/div$Hexp)
qqline((div$Hexp - div$Hobs)/div$Hexp)
t.test((div$Hexp - div$Hobs)/div$Hexp)

qqnorm(div$Hexp - div$Hobs)

hist(div$Hexp - div$Hobs)
```
```{r echo = FALSE}
plot(div$Hexp ~ div$Hobs)
qqnorm(div$Hexp - div$Hobs)
qqline(div$Hexp - div$Hobs)
```

```{r echo=FALSE}
mydata.matFst <- pairwise.fst(MyData,res.type="matrix")
kable(mydata.matFst[1:8,1:8]) 
```

